src/main/java/uk/gov/companieshouse/itemhandler/service/EmailService.java

```diff
@@ -1,76 +1,115 @@
 package uk.gov.companieshouse.itemhandler.service;
 
-import java.time.LocalDateTime;
-import java.util.UUID;
-import java.util.concurrent.ExecutionException;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.stereotype.Service;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
 import uk.gov.companieshouse.email.EmailSend;
-import uk.gov.companieshouse.itemhandler.email.CertificateOrderConfirmation;
+import uk.gov.companieshouse.itemhandler.email.OrderConfirmation;
+import uk.gov.companieshouse.itemhandler.exception.OrderMappingException;
 import uk.gov.companieshouse.itemhandler.kafka.EmailSendMessageProducer;
 import uk.gov.companieshouse.itemhandler.logging.LoggingUtils;
 import uk.gov.companieshouse.itemhandler.mapper.OrderDataToCertificateOrderConfirmationMapper;
+import uk.gov.companieshouse.itemhandler.mapper.OrderDataToCertifiedCopyOrderConfirmationMapper;
 import uk.gov.companieshouse.itemhandler.model.OrderData;
 import uk.gov.companieshouse.kafka.exceptions.SerializationException;
 
+import java.time.LocalDateTime;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
 /**
  * Communicates with <code>chs-email-sender</code> via the ` <code>send-email</code> Kafka topic to
  * trigger the sending of emails.
  */
 @Service
 public class EmailService {
 
-    private static final String NOTIFICATION_API_APP_ID =
+    private static final String CERTIFICATE_ORDER_NOTIFICATION_API_APP_ID =
             "item-handler.certificate-order-confirmation";
-    private static final String NOTIFICATION_API_MESSAGE_TYPE =
+    private static final String CERTIFICATE_ORDER_NOTIFICATION_API_MESSAGE_TYPE =
             "certificate_order_confirmation_email";
-    /** This email address is supplied only to satisfy Avro contract. */
+    private static final String CERTIFIED_COPY_ORDER_NOTIFICATION_API_APP_ID =
+            "item-handler.certified-copy-order-confirmation";
+    private static final String CERTIFIED_COPY_ORDER_NOTIFICATION_API_MESSAGE_TYPE =
+            "certified-copy_order_confirmation_email";
```

Comment:
`certified-copy_order_confirmation_email` - does this have to contain both hyphens and underscores?

Reply:
This will be updated in the next commit.